FinSearchUnified/Subscriber/Frontend.php

```diff
@@ -195,7 +196,13 @@ public function onFrontendPostDispatch(Enlight_Event_EventArgs $args)
             return;
         }
 
-        $groupKey = Shopware()->Session()->get('sUserGroup', 'EK');
+        /** @var ContextService $context */
+        $context = Shopware()->Container()->get('shopware_storefront.context_service');
```

Comment:
Please do a click-test for Shopware 5.2. It may be that this service didn't exist back then and then we must use the old implementation.

Reply:
Click tested in Shopware 5.2.26 without issues